### ISSUE TITLE
Got WindowsError: [Error 38]  when pyautogui.click

### ISSUE THREAD

**Issue by eromoe** [asweigart/pyautogui#108]
Hi, I meet a weird error today, error message as below:

```
Traceback (most recent call last):
  File "win_client.py", line 63, in <module>
    process(item['biz'], item['url'])
  File "win_client.py", line 33, in process
    pyautogui.click(x=first_ret[0], y=first_ret[1])
  File "D:\Python27\lib\site-packages\pyautogui\__init__.py", line 362, in click
    platformModule._click(x, y, 'left')
  File "D:\Python27\lib\site-packages\pyautogui\_pyautogui_win.py", line 437, in _click
    _sendMouseEvent(MOUSEEVENTF_LEFTCLICK, x, y)
  File "D:\Python27\lib\site-packages\pyautogui\_pyautogui_win.py", line 480, in _sendMouseEvent
    raise ctypes.WinError()
WindowsError: [Error 38] 已到文件结尾。
```

Main code :
```
import win32gui
import win32api
import win32con
import time
import ctypes
import requests
import pyautogui
import datetime
import json

import traceback
from sqlqueue import Queue

key_queue = Queue('key_queue')

first_ret = (506, 211)
rel_link = (925, 517)

def process(biz, url):
    pyautogui.moveTo(x=first_ret[0], y=first_ret[1], duration=1)
    pyautogui.click(x=first_ret[0], y=first_ret[1])   #  error happen here 

    pyautogui.typewrite(url, interval=0)

    pyautogui.keyDown('enter')
    pyautogui.moveTo(rel_link[0], rel_link[1], duration=1)
    pyautogui.click(x=rel_link[0], y=rel_link[1])

if __name__ == '__main__':
    
    hwnd = win32gui.FindWindow(None, u"xxxx")

    while True:
        item = key_queue.get_nowait()
        print(item)
        if item:
            try:
                win32gui.SetForegroundWindow(hwnd)
                process(item['biz'], item['url'])
            except:
                d = traceback.format_exc()
                if 'Error 38' in d:
                    print('WindowsError: [Error 38]')
                key_queue.put(item)

        time.sleep(20)
```

sqlqueue is http://github.com/eromoe/sqlqueue

I found it only got error when `item = key_queue.get_nowait()` in while loop.

1. item is a dict, like: `{ "biz": "xxxx" , "url":"xxxx" }`

2. `item` can be print out

3.  If I use `item = { "biz": "xxxx" , "url":"xxxx" }` to replace `item = key_queue.get_nowait()`, it works.

This error is weird.

**Comment by studioj** [comment]
this line raise ctypes.WinError() is I think raising ALL windows errors (check windows eventvwr) also the ones not related to pyautogui actions... 
Not entirely sure though..

**Comment by glitchassassin** [comment]
It should only be raising errors generated by the same thread, but that could be the sqlqueue code as well.

The error message seems to translate to something like "reached the end of the file."

**Comment by asweigart** [comment]
Fixed in fe22305 (or at least, we have a workaround.)
